### PR TITLE
Set GitHub status when operator is published

### PR DESCRIPTION
**Pull request by Allda**
Github commit status is set when the release pipeline publishes operator
using IIB.

JIRA: ISV-994

### CODE REVIEW

File: ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml [hunk 1]
```diff
@@ -224,3 +227,24 @@ spec:
           value: "$(tasks.get-supported-versions.results.indices)"
         - name: container_digest
           value: "$(tasks.publish-to-ocp-registry.results.container_digest)"
+
+    - name: set-github-status-published
```
Comment (MarcinGinszt) [inline ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml:231]: Well, this task should forward any results of bundle publishing (either successful or not), retrieved from IIB after the publishing call-  Jira says, that it's blocked by ISV-993, and it's done before it.

Comment (Allda) [reply ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml:231]: I added a extra results to the publishing step and reuse it as a status argument. I am going to work on the publishing step once we have a Kerberos account.

Comment (wcheang) [reply ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml:231]: When you get to the publishing step, please make sure that the status result of the step are within the list of accepted states of the github status API: https://docs.github.com/en/rest/reference/repos#create-a-commit-status

**Review by wcheang — APPROVED** [review]

File: ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml (hunk 1 above)
Comment (Allda) [reply ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml:231]: yy, I am aware of it.

**Pull request merged**